webrender_traits/Cargo.toml

```diff
@@ -25,7 +25,7 @@ serde_derive = {version = "0.8", optional = true}
 ipc-channel = { version = "0.5.0", optional = true }
 
 [target.'cfg(target_os = "macos")'.dependencies]
-core-graphics = "0.4"
+core-graphics = "0.4.3"
```

Comment:
this is not needed, given that the logic that depends on 0.4.3 is in `webrender` and it already forces the `0.4.3` version minimum.